deploy/helm/sumologic/conf/metrics/otelcol/processors.yaml

```diff
@@ -16,7 +16,7 @@ filter/drop_unnecessary_metrics:
       - resource.attributes["job"] != "pod-annotations" and IsMatch(name, "scrape_.*")
 {{- if .Values.sumologic.metrics.dropHistogramBuckets }}
       # drop histograms we've extracted sums and counts from, but don't want the full thing
-      - IsMatch(name, "^(apiserver_request_duration_seconds|coredns_dns_request_duration_seconds|kubelet_runtime_operations_duration_seconds)$")
+      - type == METRIC_DATA_TYPE_HISTOGRAM or type == METRIC_DATA_TYPE_EXPONENTIAL_HISTOGRAM or type == METRIC_DATA_TYPE_SUMMARY or IsMatch(name, ".*_bucket")
```

Comment:
how would we send one or two histogram metrics?

Reply:
by exposing new configuration option, and adding `not isMatch` at the end of the query

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/3819

Reply:
I wouldn't play with `merge` if we can do it easily by the configuration option